fastlane/lanes/build.rb

```diff
@@ -183,7 +183,8 @@
 
     upload_build_to_testflight(
       whats_new_path: WORDPRESS_RELEASE_NOTES_PATH,
-      distribution_groups: ['Internal a8c Testers', 'Public Beta Testers']
+      distribution_groups: ['Internal a8c Testers', 'Public Beta Testers'],
+      beta_app_description_path: WORDPRESS_BETA_APP_DESCRIPTION_PATH,
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/cedd79849c6abedd9f4585fb67db2d61c92dd39a/fastlane/lanes/build.rb#L187"></span>Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call.</td>
    </tr>
  </tbody>
</table>